scripts/Util.py

```diff
@@ -2346,21 +2349,20 @@ def start(self, process, current, args, props, envs, watchDog):
         }
 
         traceFile = ""
-        if not isinstance(process.getMapping(current), JavaScriptMixin):
```

Comment:
Does this mean we're now running all JS tests with logging-to-file enabled?

Reply:
No. You still have to enable it in the test. This code was here to ignore Ice.LogFile property with the JavaScript mapping, as it was previously not supported.